### ISSUE TITLE
Add flight speed ability

### ISSUE THREAD

**Issue by i509VCB** [Ladysnake/PlayerAbilityLib#2]
I've noticed that flight speed isn't in Vanilla abilities, however since it takes a float it probably won't work out of the box to just add it right in.

**Comment by Pyrofab** [comment]
Indeed, I noticed it before but I left it aside because it had to work differently and I assumed it wasn't a big deal.
I think the way to go is using an entity attribute, I'll probably try something with that.

**Comment by i509VCB** [comment]
Maybe allow abilities to be typed like gamerules is one option?

Otherwise we maybe as well start PAL (PlayerAttributesLib) lol
